### PR TITLE
Update rex manual

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Review by iNecas — APPROVED** [review]
Not merge rights, but :+1: from rex maintainer POV

File: plugins/foreman_remote_execution/1.7/index.md [hunk 1]
```diff
@@ -0,0 +1,872 @@
+---
+layout: plugin
+title: Foreman Remote Execution 1.3 Manual
```
Comment (ekohl) [inline plugins/foreman_remote_execution/1.7/index.md]: 1.3 -> 1.7 (same a line below)

File: plugins/foreman_remote_execution/1.7/index.md [hunk 2]
```diff
@@ -0,0 +1,872 @@
...
+# 1. {{ page.title }}
+
+This plugin enables Foreman to run arbitrary commands on hosts using different
+providers. Initially only an SSH provider is supported but we plan to add more.
+Communication goes through the smart proxy so Foreman does not have to have
+direct access to the target host and can scale to control many hosts. A command
+can be customized similarly to provisioning templates or partition tables. A
+number of templates are included for running basic commands, installing
+packages, etc. For more details see the
+[JobTemplates](plugins/foreman_remote_execution/{{page.version}}/index.html#3.1JobTemplates)
```
Comment (ekohl) [inline plugins/foreman_remote_execution/1.7/index.md]: I think the URL here should be `/plugins`

File: plugins/foreman_remote_execution/1.7/index.md [hunk 3]
```diff
@@ -0,0 +1,872 @@
...
+
+By default Smart Proxy listens for http connections on port 8000 and for https
+connections on port 8443. Those ports should be opened in the firewall. Smart
+Proxy Dynflow Core listens on port 8008, but the only thing which should ever
+communicate with it is the Smart Proxy running on the same machine. This means
+it should not be needed to open this port in the firewall, assuming communication
+over loopback interface is allowed.
+
+## 1.2 Compatibility
+TODO: Check the versions
```
Comment (ekohl) [inline plugins/foreman_remote_execution/1.7/index.md]: Should we do this? :)

File: plugins/foreman_remote_execution/1.7/index.md [hunk 4]
```diff
@@ -0,0 +1,872 @@
...
+
+Each component is packaged for platforms that Foreman is officially packaged
+for. Since both Foreman and Smart Proxy plugins requires Dynflow, Ruby 1.9+ is
```
Comment (ekohl) [inline plugins/foreman_remote_execution/1.7/index.md]: By now this is probably newer. At least 2.0 I'd guess

File: plugins/foreman_remote_execution/1.7/index.md [hunk 5]
```diff
@@ -0,0 +1,872 @@
...
+When using SELinux make sure the directory and the files have correct labels
+of `ssh_home_t`. If not, restore the context:
+
+    restorecon -RvF ~foreman-proxy/.ssh
+
+Don't forget to restart Foreman, Smart Proxy, Smart Proxy Dynflow and Foreman
+tasks so plugins are loaded
+
+    service httpd restart
+    service foreman-tasks restart
```
Comment (ekohl) [inline plugins/foreman_remote_execution/1.7/index.md]: I think it's `dynflowd` nowadays

File: plugins/foreman_remote_execution/1.7/index.md [hunk 6]
```diff
@@ -0,0 +1,872 @@
...
+parameter. It's the value that you can see in ENC YAML under `classes` key.
+
+If you want to use either smart variable or parameter, use Variable input type.
+Both of these values are part of ENC YAML under `parameters` key overiding each
+other if they have same name. You have to specify the name of either parameter
+or smart variable.
+
+If you're confused about the difference between smart class parameter,
+smart variable and parameter, checkout Foreman manual
+[parameters](http://www.theforeman.org/manuals/latest/index.html#4.2.3Parameters)
```
Comment (ekohl) [inline plugins/foreman_remote_execution/1.7/index.md]: Could drop the domain here and just use `/manuals`

File: plugins/foreman_remote_execution/1.7/index.md [hunk 7]
```diff
@@ -0,0 +1,872 @@
...
+back to the form and hit Preview button.
+
+You can also specify default effective user for job template. By default a
+login that's used to run the script on target host is specified by global
+setting named **remote_execution_effective_user** but if you specify another
+login, sudo would try to change effective user to this value. You can
+also enforce using **current user** login which can be handy if your users
+have same login as their posix accounts on target hosts. Last field
+is **overridable** which determines whether job template settings can be
+overridden during job invocation. See [settings chapter](plugins/foreman_remote_execution/{{page.version}}/index.html#4.2Globalsettings)
```
Comment (ekohl) [inline plugins/foreman_remote_execution/1.7/index.md]: `/plugins`?

File: plugins/foreman_remote_execution/1.7/index.md [hunk 8]
```diff
@@ -0,0 +1,872 @@
...
+
+In this case, previewing of the template would immediately trigger destroy
+method and therefore it's wrapped in preview condition.
+
+## 4.5 Kerberos authentication
+Since version 1.3.2 Remote Execution can use Kerberos for authentication.
+In order for this to work the foreman-proxy user has to have a valid TGT.
+
+This feature can be enabled on a per-proxy basis by setting :kerberos_auth
+to true in /etc/smart_proxy_dynflow_core/settings.d/remote_execution_ssh.yml.
```
Comment (ekohl) [inline plugins/foreman_remote_execution/1.7/index.md]: Add quotes to the path and above setting?

File: plugins/foreman_remote_execution/1.7/index.md [hunk 9]
```diff
@@ -0,0 +1,872 @@
...
+Follow the [same process as Foreman](/contribute.html#SubmitPatches)
+for contributing.
+
+# 7. Links
+
+* foreman_remote_execution plugin [https://github.com/theforeman/foreman_remote_execution](https://github.com/theforeman/foreman_remote_execution)
+* foreman-tasks plugin [https://github.com/theforeman/foreman-tasks](https://github.com/theforeman/foreman-tasks)
+* smart_proxy_remote_execution_ssh [https://github.com/theforeman/smart_proxy_remote_execution_ssh](https://github.com/theforeman/smart_proxy_remote_execution_ssh)
+* smart_proxy_dynflow [https://github.com/theforeman/smart_proxy_dynflow](https://github.com/theforeman/smart_proxy_dynflow)
+* issue tracker [http://projects.theforeman.org/projects/foreman_remote_execution](http://projects.theforeman.org/projects/foreman_remote_execution)
```
Comment (ekohl) [inline plugins/foreman_remote_execution/1.7/index.md]: http -> https

**Comment by ares** [comment]
All should be resolved, I dropped compatibility info, it's no longer that important (no more Ruby 1.8) and no one was keeping it up to date anyway.

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Added some minor corrections. Feel free to merge it yourself if you agree.

**Comment by ares** [comment]
Thanks, merging.

**Pull request merged**